sympy/geometry/tests/test_line.py

```diff
@@ -749,6 +749,16 @@ def test_parameter_value():
     raises(ValueError, lambda: l.parameter_value((0, 0), t))
 
 
+def test_bisection():
+    r1 = Line3D(Point3D(1, 0, 0), direction_ratio=[0, 1, 1])
+    r2 = Line3D(Point3D(0, 1, 0), Point3D(1, 1, 1))
+
+    assert r1.intersection(r2) == [Point3D(1, 1, 1)]
+
+    r3 = r1.bisection(r2)
```

Comment:
The method you added is called `bisector`

Reply:
Fixed, thanks!